ddp-workspace/projects/study-builder-ui/src/app/components/picklist-question-block/picklist-question-block.component.spec.ts

```diff
@@ -1,25 +1,25 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
-
-import { PicklistQuestionBlockComponent } from './picklist-question-block.component';
-
-describe('PicklistQuestionBlockComponent', () => {
```

Comment:
could we fix these tests?
same about text-question-block.component.spec.ts

Reply:
Sure, but not in the PR scope.